src/pretix/plugins/sendmail/tasks.py

```diff
@@ -99,6 +102,15 @@ def send_mails_to_orders(event: Event, user: int, subject: dict, message: dict,
                 if p.attendee_email == o.email and send_to_order:
                     continue
 
+                if subevent and p.subevent_id != subevent:
+                    continue
+
+                if subevents_from and p.subevent.date_from < subevents_from:
```

Comment:
Should we include subevents that started before `subevents_from` but end after `subevents_from`? IMHO not, but just to make sure.

Reply:
That would be contrary to how the filter parameters are labeled